src/Crawler.php

```diff
@@ -183,16 +204,19 @@ public function crawlSite( string $static_site_path ) : void {
                         $page_hash = md5( (string) $status_code );
                     }
 
+                    $write_contents = true;
+
                     if ( $use_crawl_cache ) {
                         // if not already cached
```

Comment:
Should this comment should be `if already cached`? or simply removed?

Reply:
Thanks for your comment.
`$write_contents` should only be `false` if the cache hash is found in the db (at line 213), so it should not be removed.

Reply:
sorry, still having trouble understanding this clearly (probably my mind just not working right!)

the comment `// if not already cached` is above the line `if ( CrawlCache::getUrl( $root_relative_path, $page_hash ) ) {`, which is why it confuses me, `CrawlCache::getUrl` being `true` would mean is *is* cached, right?

Once I get it in my head better, I can try to write an integration test to exercise this area.